hw/hw4.py

```diff
@@ -0,0 +1,363 @@
+import os.path
+import sys
+
+import numpy as np
+import pandas as pd
+import plotly.express as px
+import plotly.graph_objects as go
+import statsmodels.api as sm
```

Comment:
statsmodels returns "module not found" error

Reply:
try this again, I was an idiot and didn't update the reqs fully. it should still run to completion :crossed_fingers: just without the xdg-open issue (still looking at that) but you can open the html output manually (it will be in the plots folder) for now

Reply:
@jcurry9813 FYI, the latest commit should fix the `open` issue

Reply:
both issues fixed, sorry my mac is making life difficult lol